### ISSUE TITLE
Replace Twitter references in the docs with more appropriate alternatives

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#12483]
### Pertinent section of the Wagtail docs

Our official tutorial, embeds docs, third-party tutorials.

### Details

While we’re X users, we’ve decided to move the bulk of our activity to other platforms ([see our announcement](https://x.com/WagtailCMS/status/1834237886285103507)). We have a few Twitter references which are out of date as the platform’s changed names, and we’d now rather switch to showcasing networks that are more relevant for us and Wagtail users (for example Facebook or LinkedIn).

### Working on this

Anyone can contribute to this. View our [contributing guidelines](https://docs.wagtail.org/en/latest/contributing/index.html), add a comment to the issue once you’re ready to start.

There are a few references, so this could be separate PRs from different people:

- [ ] OEmbed docs examples
- [ ] Official tutorial
- [ ] Site settings examples

**Comment by thibaudcolas** [comment]
@ayaan-qadri nope :)

**Comment by ayaan-qadri** [comment]
Hey @Srishti-j18, I have been working on this issue, but as I’m still new to Wagtail, taking time to read and understand the documentation and codebase. Are you planning to resolve this one? If so, I’ll happily move on to another issue.

**Comment by Srishti-j18** [comment]
> Hey [@Srishti-j18](https://github.com/Srishti-j18), I have been working on this issue, but as I’m still new to Wagtail, taking time to read and understand the documentation and codebase. Are you planning to resolve this one? If so, I’ll happily move on to another issue.

Hi @ayaan-qadri ,
Yes, I am planning to work on this issue, but since you are also working on it, we can collaborate and make changes together. We can also discuss it on Slack for better judgment, as this task requires suggestions from different people to ensure meaningful improvements.

**Comment by ayaan-qadri** [comment]
Sure thing @Srishti-j18, Glad to collaborate and discuss over slack, I'm with same name as github on slack.

**Comment by ayaan-qadri** [comment]
Hey @thibaudcolas, So after reading and understating docs and bit of codebase, I can now get into this and these are the files possibly needs to change (@Srishti-j18 lemme know which files are you gonna be edit) :

![Image](https://github.com/user-attachments/assets/045fe59a-7bc5-49f5-8485-61fec95e0d1a)

I'm bit confused, like should i also need to change 'Twitter' to 'X' for example in [this tutorial](https://docs.wagtail.org/en/stable/tutorial/create_footer_for_all_pages.html#create-navigation-settings)

```twitter_url = models.URLField(verbose_name="Twitter URL", blank=True)```
to
```x_url = models.URLField(verbose_name="X URL", blank=True)```

or will this be replaced by appropriate alternatives ?

**Comment by Srishti-j18** [comment]
> ([@Srishti-j18](https://github.com/Srishti-j18) lemme know which files are you gonna be edit) :

Hey, I was a bit busy, so I haven't started editing yet. You can choose the first files you want to edit, and I'll work on the others.

**Comment by Srishti-j18** [comment]
> I'm bit confused, like should i also need to change 'Twitter' to 'X' for example in [this tutorial](https://docs.wagtail.org/en/stable/tutorial/create_footer_for_all_pages.html#create-navigation-settings)
> 
> `twitter_url = models.URLField(verbose_name="Twitter URL", blank=True)` to `x_url = models.URLField(verbose_name="X URL", blank=True)`
> 
> or will this be replaced by appropriate alternatives ?

I think we can replace it with a LinkedIn URL .It might be the better option for portfolio site.

**Comment by ayaan-qadri** [comment]
> I think we can replace it with a LinkedIn URL .It might be the better option for portfolio site.

Both works for portfolio site, but i think this is not specific to what the issue as it is more specific on wagtail itself, not the tutor which is for personal use case (it is portfolio site in tutorial).

> While we’re X users, we’ve decided to move the bulk of our activity to other platforms ([see our announcement](https://x.com/WagtailCMS/status/1834237886285103507)). We have a few Twitter references which are out of date as the platform’s changed names, and we’d now rather switch to showcasing networks that are more relevant for us and Wagtail users (for example Facebook or LinkedIn).

We just have to wait until @thibaudcolas to reply on this.

**Comment by hkx05** [comment]
Hey @ayaan-qadri and @Srishti-j18 I would also love to collaborate and work with you guys on this. I am also pretty new to wagtail so I'm also figuring out the codebase.

**Comment by thibaudcolas** [comment]
I’ve assigned this to @ayaan-qadri for the time being. Let’s leave them time to figure out which aspects of this they want to work on, before other people attempt to work on this. There are plenty other interesting issues to work on so we don’t all need to rush for this one.

**Comment by thibaudcolas** [comment]
Thank you for picking this up @ayaan-qadri! As far as I can tell you’ve updated all the relevant Twitter references in the docs with more relevant options, so I’ve closed this now.

**Comment by ayaan-qadri** [comment]
Yes @thibaudcolas, most references have been changed, but there are still some Twitter mentions. I think they are meant to be there, though (lemme know if we should change it too):

## `releses/4.0.md`:
![Image](https://github.com/user-attachments/assets/4ad84723-2036-4d59-812c-42393116000d)
https://docs.wagtail.org/en/stable/releases/4.0.html#other-features


## `reference/settings.md`
![Image](https://github.com/user-attachments/assets/612216dd-b640-4caf-9352-8c22309e0b39)
https://docs.wagtail.org/en/stable/reference/settings.html#embeds


## `releses/2.14.1.rst`
![Image](https://github.com/user-attachments/assets/594e581f-2811-4202-9858-bd94888f3b2d)
https://docs.wagtail.org/en/stable/reference/settings.html#embeds

## `advanced_topics/third_party_tutorials.md`
![Image](https://github.com/user-attachments/assets/99ceba20-7cd1-4f59-8ba6-4babd3578bcf)
https://docs.wagtail.org/en/stable/advanced_topics/third_party_tutorials.html

**Comment by thibaudcolas** [comment]
thank you @ayaan-qadri, yes I think all those references are fine.

**Comment by gasman** [comment]
@thibaudcolas I think the one in reference/settings.md could perhaps be changed to Reddit to match the same sentence in [advanced_topics/embeds.md](https://github.com/wagtail/wagtail/pull/12516/files#diff-0684bc3d17bf13e8fd1fa2c3b8c4271b6f43de226c444cd4e466e6570521e489), seeing as Twitter embeds apparently don't work any more (as reported by @cnk [on Slack](https://wagtailcms.slack.com/archives/C81FGJR2S/p1729780819899209?thread_ts=1729779240.534859&cid=C81FGJR2S))?

Agreed that the release notes and third-party tutorials don't need changing.

**Comment by ayaan-qadri** [comment]
## @gasman before making this change: 
![Image](https://github.com/user-attachments/assets/ed420ec4-92e2-4732-87e5-63d1359d95ed)

## i tested if tweeter's embeds still work or not it was working and again for surety i freshly docker setup to check is it still working:
![Image](https://github.com/user-attachments/assets/1bcf89f0-8c60-4e6a-b167-96d9aca4e2ac)

## 404.html:
![Image](https://github.com/user-attachments/assets/e4b7fc65-a273-4293-a7f7-a1644229af66)

# Note:
1. when i was trying to hit any random route it was throwing django 404 not the wagtail
2. I do not think that this happening because of old embeds (embedded when it was tweeter, not X) because still the url structure is same (post > three dots > Embed post)
